rules/windows/execution_via_hidden_shell_conhost.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/17"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects suspicious console activity which could be indicative of code injection. Verify parent process execution details
+such as network connections and file writes for context verification.
```

```suggestion
Detects when the Console Window Host (conhost.exe) process is spawned by a suspicious parent process, which could be indicative of code injection.
```

I think it'd help analysts with triaging this alert if you added a sentence to explain why an attacker would carry out this technique.

I think it makes more sense for the following sentence to go in the `notes` field of the rule, as it pertains to triaging the alert.

> Verify parent process execution details such as network connections and file writes for context verification.